onvm/onvm_mgr/onvm_nf.c

```diff
@@ -188,11 +188,20 @@ onvm_nf_start(struct onvm_nf_info *nf_info) {
                 return 1;
         }
 
+        if (nf_info->service_id >= MAX_SERVICES) {
+                // Service ID must be less than MAX_SERVICES and greater than 0
+                nf_info->status = NF_SERVICE_MAX;
+                return 1;
+        }
+
+
```

Comment:
As we're fixing bugs I think we can address another 🐛.
If we spawn more than MAX_NFS_PER_SERVICE NFs with service id X we're going to crash the manager when accessing the `nf_per_service_count` array later on.

Basically add another check that exits if  `nf_per_service_count[nf_info->service_id] >= MAX_NFS_PER_SERVICE`, Add the new error type as well that was a good idea.

Reply:
Sounds good, I added that check and created a new error type (8) for this scenario.